src/attr/_make.py

```diff
@@ -484,6 +487,9 @@ def _transform_attrs(cls, these, auto_attribs, kw_only, collect_by_mro):
         for attr_name, ca in ca_list
     ]
 
+    if field_transformer is not None:
```

Comment:
One quick (but important) thing: why are you calling this so early? I would put it at the very end. There's no reason to ignore inherited attributes, is there?

BTW another reason why calling it on all attrs is much better: the transformer has to power to add or remove attributes if it wants to! 🎉

Reply:
Do you mean like this`
```python
    if field_transformer is not None:
        attrs = field_transformer(cls, attrs)
    return _Attributes((attrs, base_attrs, base_attr_map))
```

Reply:
> BTW another reason why calling it on all attrs is much better: the transformer has to power to add or remove attributes if it wants to! 🎉

The docs contain an example where I use this hook to remove all float fields from a class – because, who likes floats? 🤪

Reply:
Seems to work. I also added a test for this.